tests/qos/qos_sai_base.py

```diff
@@ -993,7 +994,7 @@ def dutConfig(
 
         dutTopo = "topo-"
 
-        if dutAsic == "gb" and topo == "t2":
+        if dutAsic == "gb" and "t2" in topo:
```

Comment:
```
        if dutAsic == "gb" and "t2" in topo:
            if get_src_dst_asic_and_duts['src_asic'] == \
                    get_src_dst_asic_and_duts['dst_asic']:
                dutTopo = dutTopo + "any"
            else:
                dutTopo = dutTopo + topo
```
this part is missed in 202305 branch, wonder if need to filed pr to add back to 202305 branch?

Reply:
Yes, it will be needed in 202305, but this is a minor issue - the code change was needed to support t2-min topology. If no-one is running any other T2 topologies, we will not see a problem.

Reply:
Yes, we need all these to be ported to 202305.